src/Twig/Extension/UserExtension.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Twig\Extension;
+
+use App\Domain\SkillSetDomain;
+use Twig\Environment;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+final class UserExtension extends AbstractExtension
+{
+    private SkillSetDomain $skillSetDomain;
+    private Environment $twig;
+
+    public function __construct(SkillSetDomain $skillSetDomain, Environment $twig)
+    {
+        $this->skillSetDomain = $skillSetDomain;
+        $this->twig = $twig;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFilters(): array
+    {
+        return [
+            new TwigFilter('skillBadge', [$this, 'formatBadge'], ['is_safe' => ['html']]),
+        ];
+    }
+
+    public function formatBadge(string $skill, bool $markAsImportant = true): string
```

Comment:
why not using the `importantSkillsToDisplay` value here instead of passing a boolean ?